content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -62,7 +62,7 @@ To manually enable IPv4 packet forwarding:
 
 ```bash
 # sysctl params required by setup, params persist across reboots
-cat <<EOF | sudo tee /etc/sysctl.d/k8s.conf
+sudo tee /etc/sysctl.d/k8s.conf <<EOF
```

Comment:
nit: ```suggestion
echo 'net.ipv4.ip_forward = 1' | sudo tee /etc/sysctl.d/k8s.conf
```

Reply:
Well then. Let's do it right in the right way 😄 

```suggestion
sudo tee /etc/sysctl.d/k8s.conf <<<'net.ipv4.ip_forward = 1'
```